zio-http/src/main/scala/zhttp/http/HttpApp.scala

```diff
@@ -2,76 +2,56 @@ package zhttp.http
 
 import zio.ZIO
 
-final case class HttpApp[-R, +E](asHttp: Http[R, E, Request, Response[R, E]]) extends AnyVal {
-
-  /**
-   * Converts a failing Http into a non-failing one by handling the failure and converting it to a result if possible.
-   */
-  def silent[R1 <: R, E1 >: E](implicit s: CanBeSilenced[E1, Response[R1, E1]]) = HttpApp(
-    asHttp.catchAll(e => Http.succeed(s.silent(e))),
-  )
-
-  /**
-   * Combines two HttpApps into one.
-   */
-  def +++[R1 <: R, E1 >: E](other: HttpApp[R1, E1]) = HttpApp(asHttp +++ other.asHttp)
-
-  /**
-   * Evaluates the app and returns an HttpResult that can be resolved further
-   */
-  private[zhttp] final def execute(r: Request) = asHttp.execute(r)
-}
-
```

Comment:
We can add an `apply` function to support backward compatibility?

Reply:
Fixed in 5c90fab.

I also added a deprecated `asHttp` to the syntax to make it backwards compatible as well.